spec/unit/puppet/provider/conf_setting/ruby_spec.rb

```diff
@@ -194,6 +194,51 @@ def validate_file(expected_content,tmpfile = tmpfile)
       EOS
       )
     end
+
+    it "should convert a scalar key to a single element array when type is set" do
+      File.open(tmpfile, 'w') do |fh|
+        fh.write('ennui: yes')
+      end
+      validate_file('ennui: yes')
```

Comment:
Just a nitpick, but we probably don't need to validate the file, since it's just testing the behavior of `write`, right?
Similar thing in the other test